src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -1242,6 +1265,45 @@ def cli_cosmosdb_network_rule_list(client, resource_group_name, account_name):
     cosmos_db_account = client.get(resource_group_name, account_name)
     return cosmos_db_account.virtual_network_rules
 
+def cli_cosmosdb_identity_show(client, resource_group_name, account_name):
+    """ Show the identity associated with a Cosmos DB account """
+    
+    cosmos_db_account = client.get(resource_group_name, account_name)
+    return cosmos_db_account.identity
+
+def cli_cosmosdb_identity_assign(client,
+                                 resource_group_name,
+                                 account_name):
+    """ Show the identity associated with a Cosmos DB account """
+
+    existing = client.get(resource_group_name, account_name)
+
+    if ("SystemAssigned" in existing.identity.type):
+        return existing.identity
+
+    identity = ManagedServiceIdentity(type='SystemAssigned')
+    params = DatabaseAccountUpdateParameters(identity=identity)
+    async_cosmos_db_update = client.update(resource_group_name, account_name, params)
+    cosmos_db_account = async_cosmos_db_update.result()
+    cosmos_db_account = client.get(resource_group_name, account_name)  # Workaround
+    return cosmos_db_account.identity
+
+def cli_cosmosdb_identity_remove(client,
+                                 resource_group_name,
+                                 account_name):
+    """ Show the identity associated with a Cosmos DB account """
+
+    existing = client.get(resource_group_name, account_name)
+
+    if (not "SystemAssigned" in existing.identity.type):
```

Comment:
```suggestion
    if ("None" in existing.identity.type):
```
Why not check `None` since you may support other identity types in near future and `not "SystemAssigned"` can't cover all

Reply:
Done! Thank you for the hint!